.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I'd remove .vscode launch assets -- the pre-release/may release of the C# Dev Kit has a debugger launcher support for Aspire and children processes.